README.md

```diff
@@ -0,0 +1,103 @@
+# COVID-19-data-app
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+The Covid-19 data-app collects information from API and present data for the novel coronavirus.
+
+<img src="https://raw.githubusercontent.com/sentayhu19/COVID-19-data-app/features/src/images/details.png" width="45%" align="right" />
+<img src="https://raw.githubusercontent.com/sentayhu19/COVID-19-data-app/features/src/images/home.png" width="45%" />
+
+
+
+
+## Live demo comming soon
+
+[Live Demo Link on Netlify](https://covid-19-data-app1.netlify.app/)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+
+### Prerequisites
+
+- VScode or other editor
+
+### Install
+
+#### Cloning the project
+
+ git clone https://github.com/sentayhu19/COVID-19-data-app <Your-Build-Directory>
+``` 
+- cd bookstore
```

Comment:
- [ ] Kindly update the app name which comes after `cd ..` command from `bookstore ` to `COVID-19-data-app`.